lib/dom.js

```diff
@@ -1285,6 +1289,57 @@ declare class TrackDefault {
   kinds: Array<string>;
 }
 
+declare class NodeFilter {
+  static SHOW_ALL: number;
+  static SHOW_ATTRIBUTE: number;
+  static SHOW_CDATA_SECTION: number;
+  static SHOW_COMMENT: number;
+  static SHOW_DOCUMENT: number;
+  static SHOW_DOCUMENT_FRAGMENT: number;
+  static SHOW_DOCUMENT_TYPE: number;
+  static SHOW_ELEMENT: number;
+  static SHOW_ENTITY: number;
+  static SHOW_ENTITY_REFERENCE: number;
+  static SHOW_NOTATION: number;
+  static SHOW_PROCESSING_INSTRUCTION: number;
+  static SHOW_TEXT: number;
+  static FILTER_ACCEPT: number;
+  static FILTER_REJECT: number;
+  static FILTER_SKIP: number;
+  // should return an enum. one of:
+  // NodeFilter.FILTER_ACCEPT or
+  // NodeFilter.FILTER_REJECT or
+  // NodeFilter.FILTER_SKIP
+  acceptNode(node: Node): number;
+}
+
+declare class NodeIterator {
+  root: Node;
+  whatToShow: number;
+  filter: NodeFilter;
```

Comment:
I think this should be `NodeFilter | null`, per [the spec](https://dom.spec.whatwg.org/#interface-nodeiterator)